webdriver/tests/bidi/browsing_context/load/load.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from webdriver.error import TimeoutException
+
+from tests.support.sync import AsyncPoll
+from .. import assert_navigation_info
+
+pytestmark = pytest.mark.asyncio
+
+CONTEXT_LOAD_EVENT = "browsingContext.load"
+
+
+async def test_not_unsubscribed(bidi_session, inline, top_context):
+    await bidi_session.session.subscribe(events=[CONTEXT_LOAD_EVENT])
+    await bidi_session.session.unsubscribe(events=[CONTEXT_LOAD_EVENT])
+
+    # Track all received browsingContext.load events in the events array
+    events = []
+
+    async def on_event(method, data):
+        events.append(data)
+
+    remove_listener = bidi_session.add_event_listener(CONTEXT_LOAD_EVENT, on_event)
+
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url
+    )
+
+    wait = AsyncPoll(bidi_session, timeout=0.5)
+    with pytest.raises(TimeoutException):
+        await wait.until(lambda _: len(events) > 0)
+
+    remove_listener()
+
+
+async def test_subscribe(bidi_session, inline, new_tab, wait_for_event):
+    # Unsubscribe in case a previous tests subscribed to the event
+    await bidi_session.session.unsubscribe(events=[CONTEXT_LOAD_EVENT])
+
+    await bidi_session.session.subscribe(events=[CONTEXT_LOAD_EVENT])
+
+    on_entry = wait_for_event(CONTEXT_LOAD_EVENT)
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"], url=url)
```

Comment:
@lutien would you still mind adding the proposed test when opening a new tab and window and waiting for the initial `load` event?

Reply:
sure, added

Reply:
Thanks. Actually I'm unsure if we should even parameterize the method for both `tab` and `window`. Maybe it make sense?

Reply:
Yes, I think it makes sense. Added it.

Reply:
Thanks!